### ISSUE TITLE
Design Intent - Accessibility Feedback - Login.gov adoption team, DS Logon Grandfathering & Inherited Proofing

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#53333]
## VFS acceptance criteria
- [x] Let Platform know when the **Must** feedback has been incorporated 
- [x] Leave any comments for feedback you decide _not_ to take
- [x] VFS team closes the ticket

## Thoughts/questions
- I really like how you've presented these workflows! It's clear there's a ton of thought that went into them and I really appreciate it.
- I would very much encourage you to reach out to the sitewide content, accessibility, and IA team (CAIA) for help with accessibility throughout your product development. The accessibility part of CAIA is new and still onboarding, but once they're up and running they should be able to help a ton.

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**
- I didn't think to mention this in the meeting, but as you move forward with your design be aware of maintaining material honesty -- namely, links go places (eg. navigate the browser to a new URL) and should be visually styled as links, and buttons do things (eg. open/close a modal on the same page) and should be visually styled as buttons. Various assistive technologies handle links and buttons differently, and material honesty helps avoid some potentially difficult user interactions.    
   
   But this can be a tricky design challenge when you have a primary action that's navigational and a secondary action that's more appropriately a button, like "go to Login.gov to create an account" (probably a link?) and "dismiss this modal for now" (button). Finding a way to maintain material honesty while also presenting appropriate visual weight to those options is something several teams are struggling with and hasn't really been solved yet in the design system. As you build your prototypes, definitely reach out to @humancompanion-usds for ideas and/or share your ideas with him if you've got a good way of solving this problem!

**Should:**
- I would recommend working with an accessibility specialist to do a high-level audit of the Login.gov account creation process and identity proofing process. Everything on Login.gov is outside your control, but knowing what accessibility barriers may exist may inform your content choices, eg. how to frame the upgrade-to-IAL2 interaction. Your content could warn users about interaction requirements/known compatibility issues, give time-to-completion estimates, or otherwise help disabled users decide whether they want to complete the step now or later (or not at all).
- Shane mentioned testing with assistive technology users and I wholeheartedly agree. The CAIA team should be able to help you here, and I would also encourage you to ask questions/share back to the broader research practice via #inclusive-research in Slack.

**Consider:**

## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Remove the CC-Request label (only if you created the ticket using the New Issue link from the Collaboration Cycle Reviews board)
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by MOStudioCarolyn** [comment]
Chatted with @humancompanion-usds about options for links/buttons in modals![Screen Shot 2023-03-07 at 10.56.37 AM.png](https://images.zenhubusercontent.com/62598d5028d5ea49e32f0d98/176945d6-b90c-497a-be01-bff8b446b314)
